main.css

```diff
@@ -407,11 +609,195 @@ input[type=submit] {
   padding-top: 20px;
   padding-bottom: -40px;
   margin-left: -10px;
-  border-top: 1px black solid;
   display: inline-block;
   text-align: center;
 }
 
 .languages_list1 {
   text-align: center;
 }
+
+/* grid desktop */
+.grid-container {
+  display: grid;
+  padding: 10px;
+  grid-template-columns: 1fr 1fr 1fr;
+  gap: 20px;
+  max-width: 1200px;
+  margin: 20px auto;
+}
+
+article.featured {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 60px;
+  align-items: center;
+  border-bottom: 1px solid #bbc7c7;
+  padding-bottom: 40px;
+  padding-right: 10%;
+  padding-left: 10%;
+}
+
+.featured2 {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 60px;
+  align-items: center;
+  padding-bottom: 40px;
+  padding-right: 10%;
+  padding-left: 10%;
+}
+
+/* Responsive */
+@media screen and (max-width: 980px) {
```

Comment:
- [ ] I kindly suggest that you make your project styling mobile-first. That would mean that you need to use `min-width` to style for desktops and not `max-width` to style for mobile.